aws/iam_test.go

```diff
@@ -0,0 +1,41 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListIamUsers(t *testing.T) {
+	t.Parallel()
+
+	region, err := getRandomRegion()
+	if err != nil {
+		assert.Fail(t, errors.WithStackTrace(err).Error())
+	}
```

Comment:
NOTE: we've been converting these checks to `require.NoError(t, err)`, so that it short circuits the test when there is an error. Also reduces these 3 line checks to 1 line.